driver/CBAInferFusion_driver.hpp

```diff
@@ -601,7 +601,7 @@ int CBAInferFusionDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
     else
         out_sz = in_sz; // This is for N+A so the output is the same as the input size
 
-    if(miopen::IsEnabled(ENV(MIOPEN_DRIVER_PAD_BUFFERS_2M)))
+    if(env::enabled(MIOPEN_DRIVER_PAD_BUFFERS_2M))
```

Comment:
[Question][Quality] @CAHEK7 @apwojcik Do we really want to change from CamelCase to snake_case for functions?

Reply:
The problem is that we already have everything messed up. Let's recall `miopen::deref`, `miopen::rms_range`, nomen illis legio.